src/renderer/src/services/TranslateService.ts

```diff
@@ -55,7 +54,7 @@ async function fetchTranslate({ content, assistant, onResponse }: FetchTranslate
 
   const params: CompletionsParams = {
     callType: 'translate',
-    messages: content,
+    messages: '',
```

Comment:
不确定空消息是否会在部分场景下出问题，先随便塞点东西

```suggestion
    messages: 'do',
```